tests/ticket2986.py

```diff
@@ -46,8 +46,10 @@ def test(self):
 
         new = ccds.addNew()
         new.set("id", 0)
-        new.set("bbox.min", afwGeom.Point2I(0,0))
-        new.set("bbox.max", afwGeom.Point2I(1024,1024))
+        new.set("bbox_min_x", 0)
+        new.set("bbox_min_y", 0)
+        new.set("bbox_max_x", 1024)
+        new.set("bbox_max_y", 1024)
 
```

Comment:
Is there a nicer way to do this, e.g. using a bbox key and setting it from a bbox (I realize those are a pain to construct, but I think it would still be clearer and more robust than setting 4 fields independently).